### ISSUE TITLE
Language to Network: Conditional Parameter Adaptation with Natural Language Descriptions 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#35]
# TL;DR

A study of zero-shot image classification of unknown categories with using explanatory text processed by pre-trained BERT. The performance is tested on four datasets, with a significant improvement in accuracy compared to previous studies. The evidence can be visualized at the token level.
![image](https://user-images.githubusercontent.com/24524018/89734105-fba30000-da94-11ea-90e1-fed06aaa5852.png)

# Why it matters:

# Paper URL
https://www.aclweb.org/anthology/2020.acl-main.625/

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Tian Jin, Zhun Liu, Shengjia Yan, Alexandre Eichenberger, Louis-Philippe Morency
- IBM Thomas J. Watson Research Center 
- Microsoft
- Tandon School of Engineering 
- Carnegie Mellon University 

# Methods

# Results

# Comments
ACL2020
